cmux-tui/scripts/iroh-docker-demo/attach-once.py

```
@@ -0,0 +1,282 @@
+        handshake = {
+            "protocol": "cmux.machine-provider",
+            "version": 1,
+            "role": "transport",
+            "token": token,
+            "ticket": ticket,
+        }
+        self.process.stdin.write(json.dumps(handshake) + "\n")
+        self.process.stdin.flush()
+        result = json.loads(self.reader.readline(30.0, "transport handshake"))
+        if not result.get("accepted"):
+            raise RuntimeError(f"transport handshake rejected: {result}")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)